src/components/definition-list/dd.component.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledDd } from './definition-list.style';
+
+const Dd = ({
+  mb = 2,
+  children = 'Content for associated label'
```

Comment:
I think `children` should be `isRequired` rather than have a default

Reply:
Agreed and consider it done.

Reply:
✅